src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/security.yml

```diff
@@ -0,0 +1,85 @@
+admin_security:
+    path: /
+    methods: [GET]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\Security:index'
+        _legacy_controller: AdminSecurity
+        _legacy_link: AdminSecurity
+
+admin_security_general_save:
+    path: /general
+    methods: [POST]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\Security:processGeneralForm'
+        _legacy_controller: AdminSecurity
+        _legacy_link: AdminSecurity
+
+admin_security_sessions_customer:
```

Comment:
Just a detail, but the route name doesn't reflect the fact that it's a listing, or a search you could use one of the following:
- `admin_customer_sessions_index`
- `admin_customer_sessions_list`

Or if you want to keep the security domain:
- `admin_security_customer_sessions_index`
- `admin_security_customer_sessions_list`